packages/webapp/src/containers/MediaWithAuthentication/index.jsx

```diff
@@ -21,11 +21,15 @@ export function MediaWithAuthentication({
     const fetchMediaUrl = async () => {
       try {
         subscribed = true;
-        const url = new URL(fileUrl);
-        url.hostname = 'images.litefarm.workers.dev';
-        const response = await fetch(url.toString(), config);
-        const blobFile = await response.blob();
-        subscribed && setMediaUrl(URL.createObjectURL(blobFile));
+        if (import.meta.env.VITE_ENV === 'development') {
+          subscribed && setMediaUrl(fileUrl);
+        } else {
+          url.hostname = 'images.litefarm.workers.dev';
+          const url = new URL(fileUrl);
```

Comment:
These two lines should be swapped, right...?

Reply:
Oh goodness 🤦‍♀️ Thank you